.github/workflows/release_staged.yml

```diff
@@ -121,6 +127,7 @@ jobs:
   publishing-to-s3-windows:
     name: Publish windows artifacts into s3 prod bucket
     runs-on: ubuntu-20.04
+    if: ${{ (github.event.inputs.assetsType == 'all' || github.event.inputs.assetsType == matrix.assetsType) && (github.event.inputs.targzOS == 'all' || github.event.inputs.targzOS == matrix.targzOS) }}
```

Comment:
shouldn't this and docker below just ignore the targzOs option?

Reply:
Also I think you can't use matrix variables in job definition. I remember having issues with that in the past. Probably they changed workflow, so better to test it with testing bucket.